practice-two/src/pages/Analytics/Analytics.jsx

```diff
@@ -145,10 +152,29 @@ const Analytics = () => {
         )}
         {isError && <Toast message={MESSAGES.GET.ERRORS.API_FAILED} />}
       </div>
+
+      {/* Show information of selected customer */}
       {selectedCustomer && isShowProfileInfo && (
         <ProfileInfo selectedCustomer={selectedCustomer} />
       )}
-      {isShowForm && <FormValidation onShowForm={handleShowForm} />}
+
+      {/* Show form to create customer */}
+      {isShowForm && (
+        <FormValidation
+          handleShowForm={handleShowForm}
+          setCreateResult={setCreateResult}
+        />
+      )}
+
+      {/* Show Success Toast component when creating customer successfully */}
+      {createResult === 'success' && (
+        <Toast message={MESSAGES.GET.SUCCESSES.ADD_SUCCESSED} />
```

Comment:
should render Toast message on parent page.
`createResult` what doest it mean?

Reply:
I move Toast from Analytics component to App component
![image](https://github.com/datdragon777/React-Training/assets/74806826/491a9555-1373-4871-87b8-279f70969389)